dell-restore-system

```diff
@@ -0,0 +1,89 @@
+#! /usr/bin/python3
+# -*- coding: utf-8 -*-
+#
+# Dell Recovery Restore system command line
+#
+# Copyright (C) 2017 Dell Inc,
+#   Author: Mario Limonciello/Jude Hung
+#
+# This is free software; you can redistribute it and/or modify it under
+# the terms of the GNU General Public License as published by the Free
+# Software Foundation; either version 2 of the License, or at your option)
+# any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this application; if not, write to the Free Software Foundation, Inc., 51
+# Franklin St, Fifth Floor, Boston, MA  02110-1301  USA
+##################################################################################
+
+from Dell.recovery_common import (find_partition, check_version, DBUS_INTERFACE_NAME,
+                                  DBUS_BUS_NAME, dbus_sync_call_signal_wrapper,
+                                  PermissionDeniedByPolicy)
+import optparse
+import os
+import subprocess
+import sys
+#Translation support
+from gettext import gettext as _
+
+usage = '%prog [options]'
+parser = optparse.OptionParser(usage=usage)
+parser.set_defaults(
+    version=''
+    )
+parser.add_option('-c', '--check-version', dest='checkversion', action='store_true',
+                  help='Show the version information.')
+parser.add_option('-y', '--yes', dest='yes', action='store_true',
+                  help="Restore the system without user's confirmation.")
+(options, args) = parser.parse_args()
+
+if __name__ == '__main__':
+    if options.checkversion:
+        print("Version: %s" % check_version())
+    else:
+        if not options.yes:
+            confirm = input('Would you want to restore the system?(y/n)')
```

Comment:
nitpick; place a space between ? and (

Reply:
Do you want this string translated?  You should invoke with _ if so.

Reply:
Sure, I'll add a new space between ? and (.

This string is not currently available in *.po.  If I'd like to get this string translated, we need to add a new msgid into *.po.  However, I could help to translate this new string in zh_TW.po, but for other languages, how do you usually get the new string translated? Google Translate?

Reply:
You should add the file to https://github.com/dell/dell-recovery/blob/master/po/POTFILES.in so it can get translated.  We were relying on translations service from (Launchpad)[https://translations.launchpad.net/dell-recovery] but frankly haven't updated that in a while.  If you want to add strings for zh_TW.po please go ahead though.